pkg/security/ebpf/c/include/structs/syscalls.h

```diff
@@ -74,6 +74,13 @@ struct syscall_cache_t {
             struct file_t target_file;
         } rename;
 
+        struct {
+            int resource;
+            u64 rlim_cur;
+            u64 rlim_max;
+            u32 pid;
```

Comment:
This `pid` field doesn't seem to be used, we can probably remove it from this struct